### DISCUSSION TITLE
LXQt Weblate (Translations)

### DISCUSSION THREAD

**Discussion by hugok79** [lxqt/lxqt#2624] [General]
Why is it that in LXQt Weblate the Portuguese language has all the components translated and in some cases the total number of strings is not 100% as shown in the image below?

![2024-10-11_21-46-56](https://github.com/user-attachments/assets/bf660942-c99e-4ed1-9d8a-cff0a9adc26d)

**Comment by stefonarch** [comment]
I noticed that too. Basically 98%=100% in this components. No clue atm. Untranslated gives hints what is missing for <98%

![immagine](https://github.com/user-attachments/assets/cf21893a-86fd-493c-918d-a3d74476bc52)
